internal/cli/atlas/backup/exports/jobs/create.go

```diff
@@ -67,21 +67,21 @@ func (opts *CreateOpts) newExportJob() *mongodbatlas.CloudProviderSnapshotExport
 		customData = append(customData, pair)
 	}
 	createRequest := &mongodbatlas.CloudProviderSnapshotExportJob{
-		SnapshotID:     opts.snapshotID,
-		ExportBucketID: opts.exportBucketID,
-		CustomData:     customData,
+		SnapshotID:  opts.snapshotID,
+		ExportJobID: opts.exportJobID,
+		CustomData:  customData,
```

Comment:
most of these errors happened because you are not running the commit hook. Run `make link-git-hooks` in `mongodb-atlas-cli` to install it